src/main/java/com/epam/test_generator/controllers/step/request/StepUpdateDTO.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.test_generator.controllers.step.request;
+
+import com.epam.test_generator.entities.Status;
+import com.epam.test_generator.entities.StepType;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
+public class StepUpdateDTO {
+
+    @Min(value = 1)
+    private Integer rowNumber;
+
+    @Size(min = 1, max = 255)
+    private String description;
+
+    private StepType type;
+
+    private String comment;
```

Comment:
Comment is not limited by size?

Reply:
What length it has as a DB field?

Reply:
255, added annotation with corresponding restricton